app/assets/stylesheets/modules/bibliography.scss

```diff
@@ -8,18 +9,19 @@
     padding-left: 0;
   }
 
-  .bibliography-list {
+  .bibliography-list,
+  .cited-documents-list {
     margin-left: 0;
     padding-left: 0;
 
     a {
       white-space: nowrap;
     }
 
-    .bibliography-body {
+    .bibliography-body,
+    .cited-documents-body {
       margin-left: 2em;
       max-width: 80ch;
-      text-indent: -2em;
```

Comment:
I think this might be causing an issue on the bibliography view:

<img width="661" alt="screen shot 2017-10-27 at 7 41 52 pm" src="https://user-images.githubusercontent.com/1656824/32128789-e7e3e308-bb4e-11e7-9599-c3cbdd829858.png">

Could we modify the selector or classes here?